packages/@n8n/db/src/entities/credentials-entity.ts

```diff
@@ -42,6 +42,25 @@ export class CredentialsEntity extends WithTimestampsAndStringId implements ICre
 	@Column({ default: false })
 	isGlobal: boolean;
 
+	/**
+	 * Whether the credential can be dynamically resolved by a resolver.
+	 */
+	@Column({ default: false })
+	isResolvable: boolean;
+
+	/**
+	 * Whether the credential resolver should allow falling back to static credentials
+	 * if dynamic resolution fails.
+	 */
+	@Column({ default: false })
+	resolvableAllowFallback: boolean;
+
+	/**
+	 * ID of the dynamic credential resolver associated with this credential.
+	 */
```

Comment:
Can we clarify that this is many to one in the docline?

Reply:
Yes that's true. we thought it would be fine to have that, as we're not using typeorm for migration generation. But I see your point 🤔 
I think it could be possible to extends the credential entity from the module itself (no sure, have to test). Not sure if it makes sense either though